themes/default/layouts/partials/head.html

```diff
@@ -27,6 +27,32 @@
     {{ end }}
 
 
+    <!-- Events info for Google events. -->
+    {{ if .Params.event_data }}
+        <script type="application/ld+json">
+            {
+                "@context": "https://schema.org",
+                "@type": "Event",
+                "name": "{{ .Params.event_data.name }}",
+                "startDate": "{{ .Params.event_data.start_date }}",
+                "endDate": "{{ .Params.event_data.end_date }}",
+                "eventStatus": "https://schema.org/EventScheduled",
+                "eventAttendanceMode": "https://schema.org/OnlineEventAttendanceMode",
+                "location": {
+                  "@type": "VirtualLocation",
+                  "url": {{ .Params.event_data.url }}
```

Comment:
Does this need quotes?
```suggestion
                  "url": "{{ .Params.event_data.url }}"
```

Reply:
Weirdly, no. Adding quotes escapes the URL, JSON comes out clean without them.

Reply:
(I was absolutely baffled by this earlier)